lib/src/runner/configuration/args.dart

```diff
@@ -82,14 +82,14 @@ final ArgParser _parser = (() {
   parser.addOption("timeout",
       help: 'The default test timeout. For example: 15s, 2x, none',
       defaultsTo: '30s');
-  // Existing users to be notified prior to removal:
+  // This is a temporary option which will be removed in the future. Please
+  // notify the package:test authors if you are using this option so you can be
+  // notified before it is removed.
+  //
+  // Existing users:
   //   - package:build_runner
   parser.addOption('line-length',
-      help: 'Overrides the max width of lines to print.\n'
-          'This is a temporary option which will be removed in the future.'
-          'Please notify the package:test authors if you are using this option '
-          'so you can be notified before it is removed.',
-      hide: true);
+      help: 'Overrides the max width of lines to print.', hide: true);
```

Comment:
I'd move this help string to the comment as well.

Reply:
done